src/components/Recipe/Recipe.js

```diff
@@ -0,0 +1,214 @@
+import axios from "axios";
+import { useState, useContext, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import AppContext from "../../context/AppContext";
+import {
+  Box,
+  Button,
+  Grid,
+  Typography,
+  Paper,
+  Table,
+  TableBody,
+  TableContainer,
+  TableHead,
+  TableRow,
+  TableCell,
+} from "@mui/material";
+import RestaurantIcon from "@mui/icons-material/Restaurant";
+
+const Recipe = () => {
+  const [recipe, setRecipe] = useState({});
+  const [isRecipeOpen, setIsRecipeOpen] = useState();
+  const { id } = useParams();
+  const { loggedInUser } = useContext(AppContext);
+  useEffect(() => {
+    // (recipe ? setRecipe({}) : getRecipe(id));
+    const getRecipe = async (id) => {
+      console.log("ID ", id);
+      const response = await axios.get(`/v1/recipes/${id}`);
+      console.log("Response ", response.data);
+      setRecipe(response.data);
+    };
+
+    getRecipe(id);
+  }, [id]);
+
+  let recipe_user_id = recipe.user_id;
+  let isEditable =
+    loggedInUser && recipe_user_id === loggedInUser.id ? true : false;
+  let recipe_user = recipe.user;
+
+  return (
+    <Paper>
+      <Grid container direction="row">
+        <Grid
+          container
+          direction="column"
+          padding="10px"
+          justifyContent="center"
+          alignItems="center"
+          width="50%"
+        >
+          {/* Recipe title and description box. 1 of 2 items in grid container. */}
+
+          {/* Recipe Title */}
+          <Grid item>
+            <Typography
+              variant="h6"
+              component="a"
+              sx={{
+                mr: 2,
+                fontFamily: "monospace",
+                fontWeight: 700,
+                letterSpacing: ".3rem",
+                color: "inherit",
+                textDecoration: "none",
+              }}
+            >
+              {`${recipe.title}`}
+            </Typography>
+          </Grid>
+          {/* Recipe Description */}
+          <Grid item>
+            <Typography variant="subtitle1" component="a">
+              {recipe.description}
+            </Typography>
+          </Grid>
+        </Grid>
+
+        {/* Recipe Image and recipe metadata. 2 of 2 items in grid container. */}
+        <Grid
+          container
+          direction="column"
+          padding="10px"
+          width="50%"
+          justifyContent="center"
+          alignItems="center"
+        >
+          {/* Recipe Img */}
+          <Grid item>
+            <img
+              src={recipe.image_url}
+              alt={`${recipe.title}`}
+              id="recipe-image"
+              max-height="300px"
+              width="75%"
+            ></img>
```

Comment:
img tag can and should be self closing because it can't handle any children
```suggestion
            <img
              src={recipe.image_url}
              alt={`${recipe.title}`}
              id="recipe-image"
              max-height="300px"
              width="75%"
            />
```

Reply:
resolved